### PR TITLE
Write counties to db 

### PR DESCRIPTION
**Pull request by justinebreuch**
## Description

Addresses: [Complete data transformation](https://app.shortcut.com/blueconduit/story/5678/complete-data-transformation)

Writes counties to db so that all data layers can be aggregated by county. 

### New

- County handler that uses new factory. 
- Lambda function factory and abstracts away lambda function constructor

### Changed

Data import to use new lambda function factory

## Testing and Reviewing

Tested via AWS console

### CODE REVIEW

**Review by kailamjeter — APPROVED** [review]

File: cdk/src/open-data-platform/data-plane/data-import/write-county-data.handler.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      .nameWithDescription(properties.NAMELSAD)
+      // Keep JSON formatting. Post-GIS helpers depend on this.
+      .geom(JSON.stringify(value.geometry))
+      .build()
+  );
+}
+
+/**
+ * Parses S3 'cb_2021_us_county_500k.geojson' file and writes rows
+ * to water systems table in the MainCluster postgres db.
```
[the remark this answers is not in the document]
Comment (justinebreuch) [reply cdk/src/open-data-platform/data-plane/data-import/write-county-data.handler.ts:74]: Thanks

File: cdk/src/open-data-platform/data-plane/model/county_table.ts [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    this._row.state_fips = stateFips;
+    return this;
+  }
+
+  countyAnsi(countyAnsi: string): CountiesTableRowBuilder {
+    this._row.county_ansi = countyAnsi;
+    return this;
+  }
+
+  affGeoId(addGeoId: string): CountiesTableRowBuilder {
```
Comment (kailamjeter) [inline cdk/src/open-data-platform/data-plane/model/county_table.ts:77]: typo

Comment (justinebreuch) [reply cdk/src/open-data-platform/data-plane/model/county_table.ts:77]: oops

**Review by beekley — APPROVED** [review]

File: cdk/src/open-data-platform/data-plane/data-import/lambda_function_factory.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import * as lambda from 'aws-cdk-lib/aws-lambda-nodejs';
```
[the remark this answers is not in the document]
Comment (justinebreuch) [reply cdk/src/open-data-platform/data-plane/data-import/lambda_function_factory.ts]: Done!

File: cdk/src/open-data-platform/data-plane/data-import/lambda_function_factory.ts [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    environment: {
+      CREDENTIALS_SECRET: props.credentialsSecret.secretArn,
+      DATABASE_NAME: props.db,
+      RESOURCE_ARN: props.cluster.clusterArn,
+    },
+    memorySize: 512,
+    timeout: Duration.minutes(15),
+    bundling: {
+      externalModules: ['aws-sdk'],
+      nodeModules: ['stream-json', 'stream-chain'],
```
[the remark this answers is not in the document]
Comment (justinebreuch) [reply cdk/src/open-data-platform/data-plane/data-import/lambda_function_factory.ts]: Yeah agreed I think this will need to be updated in the near term!

**Pull request merged**